### DISCUSSION TITLE
Hydrogen-1.1.0

### DISCUSSION THREAD

**Discussion by cme** [hydrogen-music/hydrogen#1364] [General]
Release 1.1.0 of Hydrogen is the new stable release of Hydrogen. This release brings new features as well as many bug fixes, and is generally recommended for all users.

New features in this release include:

  - Keyboard cursor-driven pattern, song and note properties editing
  - Note selection and movement in pattern editors
  - New OSC commands (new song, open song, save song (as), quit,
      * toggle looped playback, toggle timeline, toggle Jack transport,
      * toggle Jack timebase master, add/delete timeline marker,
      * toggle song/pattern playback mode)
  - NSM support reworked
  - Deprecating JACK-session
  - Instrument main pitch shift offset
  - Custom pattern size support with representation in note values
  - Custom pan law support in mixer
  - Audio output device selection in CoreAudio (macOS) and PortAudio (Windows)

The complete list of bugs and other issues addressed by this release is available [via the Github issue tracker](https://github.com/hydrogen-music/hydrogen/milestone/5?closed=1).

<hr /><em>This discussion was created from the release <a href='https://github.com/hydrogen-music/hydrogen/releases/tag/1.1.0'>Hydrogen-1.1.0</a>.</em>

**Comment by NoAliby** [comment]
@cme 
A quick note that the Item on the Home Page http://hydrogen-music.org/ has the heading "Hydrogen 1.1.0 released", but the content covers the newer Release 1.1.1 
(Links to http://hydrogen-music.org/release-111 are correct, just the heading needs fixing)

**Reply by cme** [reply]
Thanks!
